pages/blog/houdini-and-kitql.mdx

```diff
@@ -0,0 +1,218 @@
+  date: '2022-07-04',
+  description:
+    "It's time to unveil a massive effort by the KitQL and Houdini maintainers to join forces!",
+  image: '/blog-assets/houdini-and-kitql/2022 05 25 - KitQL & Houdini.gif',
+  thumbnail: '/blog-assets/houdini-and-kitql/thumbnail.png'
+}
+
+export default withArticle(meta)
+
+## A Tale of Two Projects
+
+[SvelteKit](https://kit.svelte.dev/) is taking the web dev community by storm by providing an excellent solution
+for server-side rendered web applications and simplifying the creation of "WebApp"s
+... whatever that means 😅. While most of the common libraries for GraphQL apps do
+have svelte bindings, none of them really _spark joy_. It was hard to avoid this feeling
+that people were trying to smash a React-shaped wedge into a Svelte-shaped hole.
+Independently, [Alec](https://github.com/AlecAivazis) and [Jean-Yves](https://twitter.com/jycouet)
+set out to give the Svelte community something better and took very different approaches
+to the problem. Eventually we connected on GitHub started to discuss the pros and cons
+of each approach. During this team, one question kept popping up:
+
+> What would the best experience for using #Svelte and #GraphQL look like?
+
+It was very clear we both wanted to provide "the best" solution for using GraphQL and
+Svelte. It seemed silly to work independently since we shared this common goal
+so all we needed to do was figure out how to structure our joint efforts.
+
+## The Best of Both Worlds
+
+While at first we thought that Houdini and KitQL were solving the same problem, during
+our conversation it became clear that KitQL's scope was significantly larger than Houdini's.
+KitQL strives to provide the best possible, "batteries included" solution for fullstack
+applications built with GraphQL and SvelteKit. It achieves this by integrating powerful
+tools in the GraphQL community such as [Yoga](https://www.graphql-yoga.com/),
+[Envelop](https://www.envelop.dev/), [Modules](https://www.graphql-modules.com/),
+[Scalars](https://www.graphql-scalars.dev/), [GraphQL-ESLint](https://github.com/B2o5T/graphql-eslint), and [CodeGen](https://www.graphql-code-generator.com/)
+with its own custom client that ran in the browser. On the other hand, Houdini is entirely focused on the
+client-side of the GraphQL picture and fills a similar role to libraries like Relay, urql,
+and Apollo. With this realizaton, the path forward was clear: bring the best parts of
+the KitQL client into Houdini and integrate it with the rest of the tools in the KitQL Stack.
+
+Now users no longer have to choose between KitQL or Houdini. If you are looking for:
+
+- a GraphQL client, we recommend to use Houdini
+- building a full-stack app, we recommend KitQL _(including Houdini of course 😉)_
+
+> And it's available Today 🎉🎉🎉
+
+## How does it looks like to use GraphQL in SvelteKit?
+
+You have now two ways that you can mix and match in your application:
+
+### External Documents
```

Comment:
Not sure this title is necessary in an article (it would make sense in a documentation page though)

Reply:
Here, we want to give 2 examples of what is really key in the new way of working External Documents & Inline Documents.
Then, they will jump into the doc 👍